DepositApi.BLL/DTO/DepositDTO.cs

```diff
@@ -6,10 +6,14 @@ namespace DepositApi.BLL.DTO
 {
     public class DepositDTO
     {
+        public int Id { get; set; }
+
         public double Amount { get; set; }
 
         public int Term { get; set; }
 
         public double Percent { get; set; }
```

Comment:
Consider to change data type from `double` to `decimal`, particulary if the bussiness logic of the application handles financial operation that required data accuracy. 
For more info: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types